cuda_core/cuda/core/experimental/_module.py

```diff
@@ -215,6 +217,35 @@ def attributes(self) -> KernelAttributes:
             self._attributes = KernelAttributes._init(self._handle)
         return self._attributes
 
+    def _get_arguments_info(self, param_info=False) -> tuple[int, list[NamedTuple]]:
+        attr_impl = self.attributes
+        if attr_impl._backend_version != "new":
+            raise NotImplementedError("New backend is required")
+        arg_pos = 0
+        if param_info:
+            ParamInfo = namedtuple("ParamInfo", ["offset", "size"])
```

Comment:
Suggestion to move this to class scope, I think then it reads really well:
```
Kernel.ParamInfo
```

Reply:
Don, it does make it cleaner. pushed